test/test_tree_1d_shallowwater_wet_dry.jl

```diff
@@ -0,0 +1,396 @@
+module TestExamples1DShallowWaterWetDry
+
+# TODO: TrixiShallowWater: move any wet/dry tests to new package
+
+using Test
+using Trixi
+using TrixiShallowWater
+
+# TODO: Right now this is a local copy of the file from Trixi.jl. How can I use @trixi_testset & 
+#       @test_trixi_include from Trixi.jl?
+include("test_trixi.jl")
```

Comment:
There is no way to do this right now. We are considering creating a common TrixiBase.jl package to allow this. See https://github.com/trixi-framework/Trixi.jl/issues/1804

Reply:
Alright, then I'll leave it like this for now. 
The ToDo-comment has been removed in 8184d681910c912aed2b0b9ccb51fd820c7bde04